uint8 command
uint8 duty_cycle
uint8 direction
